src/core/components/hv-root/index.tsx

```diff
@@ -53,6 +53,7 @@ export default class Hyperview extends PureComponent<HvScreenProps.Props> {
             onParseBefore={this.props.onParseBefore}
             openModal={this.props.openModal}
             push={this.props.push}
+            refreshControl={this.props.refreshControl}
```

Comment:
I don't understand this line given the context provider wrapping the component.

Reply:
it was needed because the context was re-injected in the `hv-screen` component, using that prop. I removed the duplication, and will address the duplication for other contexts separately.
[cd658f4](https://github.com/Instawork/hyperview/pull/673/commits/cd658f4dc3c968c72925a6afdbbe0793cdde78bb)